src/rbindlist.c

```diff
@@ -63,7 +63,7 @@ SEXP rbindlist(SEXP l, SEXP usenamesArg, SEXP fillArg, SEXP idcolArg)
 
   int *colMap=NULL; // maps each column in final result to the column of each list item
   if (usenames==TRUE || usenames==NA_LOGICAL) {
-    // here we proceed as if fill=true for brevity (accounting for dups is tricky) and then catch any missings after this branch
```

Comment:
i think this is fine, an informal shorthand for "missing items/entries" that's perfectly suitable for a comment

Reply:
Undid this change as well.